shapeshift/web
# chore: consolidate consent banner and mixpanel flags, enable mixpanel everywhere

## Description



- consolidate consent banner and mixpanel flags
- enable mixpanel in all environments

## Notice




## Pull Request Type


## Issue (if applicable)



n/a

## Risk

low - tracking only



## Testing

- see that the consent banner appears once on desktop only
- see that there is no consent banner on mobile



### Engineering

👆



### Operations

👆



## Screenshots (if applicable)

see consent banner PR https://github.com/shapeshift/web/pull/3912
